plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -91,8 +101,12 @@
     private final String rootTableLocation;
     private final ConnectorPageSink insertPageSink;
     private final List<DeltaLakeColumnHandle> dataColumns;
+    private final List<DeltaLakeColumnHandle> nonSynthesizedColumns;
     private final int tableColumnCount;
     private final int domainCompactionThreshold;
+    private final Supplier<DeltaLakeCdfPageSink> cdfPageSinkSupplier;
+    private DeltaLakeCdfPageSink cdfPageSink;
```

Comment:
Nit, usually put non-final variables in a different section from the final ones. Also mark it `@nullable`